modules/custom/drupalmel_core/drupalmel_core.install

```diff
@@ -81,6 +81,7 @@ function drupalmel_core_update_7000() {
   $feeds = array(
     'Kattekrab'   => 'http://kattekrab.net/taxonomy/term/322/feed',
     'Realityloop' => 'http://realityloop.com/tags/drupal-planet/feed',
+    'Flink' => 'http://flink.com.au/taxonomy/term/1/feed',
```

Comment:
This function is a previously run hook_update_N(), and as such it will never run again. You'll need to make a new hook_update_N() function (drupalmel_core_update_7001), and as you are just adding the one node you won't need to do the foreach loop.